src/tmx/TmxUtils/src/SNMPClient.cpp

```diff
@@ -131,15 +276,14 @@ namespace tmx::utils
         // net-snmp has several methods for creating an OID object
         // their documentation suggests using get_node. read_objid seems like a simpler approach
         // TO DO: investigate update to get_node
-        if (!read_objid(input_oid.c_str(), OID, &OID_len))
-        {
-            // If oid cannot be created
+        if (!snmp_parse_oid(input_oid.c_str(), OID, &OID_len)) {
```

Comment:
It seems that they are similar, why we have a preference to this 
snmp_parse_oid method? comment still says read_objid

Reply:
I don't think it matters very much which of these methods we use as long as it works as intended and does not have any significant performance impact so I think this implementation is probably fine.